### PR TITLE
Feature/emissions widget

### PR DESCRIPTION
**Pull request by pjosh**
## Overview

- Dyanamic sentence
- Stacked chart of historical LULC and agriculture emissions
* You need to add this to your `.env` file: `CLIMATE_WATCH_API_URL=https://climate-watch.vizzuality.com/api/v1`

## Demo

captura de pantalla 2018-01-31 a las 16 17 50

### CODE REVIEW

File: app/javascript/pages/country/data/colors.json [hunk 1]
```diff
@@ -35,5 +35,8 @@
       "Mosaic of small-sized plantations": "#a6d19f",
       "Clearing/ very young plantation": "#d5b7a0"
     }
+  },
+  "emissions": {
+    "ramp": ["#073452", "#204665", "#345978", "#476c8c", "#5b80a0", "#799ab7", "#97b4cf", "#b5cfe7", "#d4ebff"]
```
Comment (edbrett) [inline app/javascript/pages/country/data/colors.json]: can we add the central color too?

File: app/javascript/pages/country/data/widgets-config.json [hunk 2]
```diff
@@ -306,5 +306,17 @@
...
     },
     "active": true
+  },
+  "emissions": {
+    "title": "Historical emissions",
+    "config": {
+      "size": "small",
+      "categories": ["land-use"],
+      "admins": ["country"],
+      "selectors": [],
```
Comment (edbrett) [inline app/javascript/pages/country/data/widgets-config.json]: You dont need the key if empty

File: app/javascript/pages/country/widget/widgets/widget-emissions/widget-emissions-selectors.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+export const getChartData = createSelector([getSortedData], sortedData => {
+  if (!sortedData || !sortedData.data.length) return null;
+
+  const { data, total } = sortedData;
+  const chartData = [];
+  data[0].emissions.forEach((item, i) => {
+    chartData.push({
+      year: item.year,
+      e1_value: item.value,
```
Comment (edbrett) [inline app/javascript/pages/country/widget/widgets/widget-emissions/widget-emissions-selectors.js]: I dont like using kebab case when we can avoid it in js but it is ok for now.

File: app/javascript/services/emissions.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+const REQUEST_URL = `${
+  process.env.CLIMATE_WATCH_API_URL
+}/emissions?gas={gas}&location={adm0}&source={source}`;
+
+export const getEmissions = ({ country }) => {
+  const url = REQUEST_URL.replace('{adm0}', country)
+    .replace('{gas}', process.env.RAILS_ENV === 'production' ? 107 : 253)
```
Comment (edbrett) [inline app/javascript/services/emissions.js]: Is the production check necessary if we always use the same CLIMATE WATCH KEY?

File: app/javascript/pages/country/widget/widgets/widget-emissions/widget-emissions-selectors.js (hunk 3 above)
Comment (pjosh) [reply app/javascript/pages/country/widget/widgets/widget-emissions/widget-emissions-selectors.js]: yeah me too... 😆  I don't know why I've used this nomenclature in this case

**Pull request merged**